Sources/SwiftSyntax/generated/raw/RawSyntaxNodesTUVWXYZ.swift

```diff
@@ -1411,7 +1411,13 @@ public struct RawTypeInitializerClauseSyntax: RawSyntaxNodeProtocol {
 @_spi(RawSyntax)
 public struct RawTypeSpecifierListSyntax: RawSyntaxNodeProtocol {
   public enum Element: RawSyntaxNodeProtocol {
+    /// A specifier that can be attached to a type to eg. mark a parameter as `inout` or `consuming`
     case simpleTypeSpecifier(RawSimpleTypeSpecifierSyntax)
+    /// A specifier that specifies function parameter on whose lifetime a type depends
+    /// - Experiment: Requires experimental feature `nonescapableTypes`.
+    #if compiler(>=5.8)
+    @_spi(ExperimentalLanguageFeatures)
+    #endif
     case lifetimeTypeSpecifier(RawLifetimeTypeSpecifierSyntax)
```

Comment:
API attributes, documentation and experimental note were omitted on the raw side.